src/getImports/packageJsonParser.js

```diff
@@ -0,0 +1,37 @@
+export function getPackages(fileName, source) {
+  const packages = [];
+
+  const lines = [];
+  source.split(/\r?\n/).forEach(function(line) {
+    lines.push(line);
+  });
+
+  const pjson = JSON.parse(source);
+
+  for (var dep in pjson.dependencies) {
```

Comment:
Should we also be scanning folks [devDependencies](https://docs.npmjs.com/files/package.json#devdependencies) and [peerDependencies](https://docs.npmjs.com/files/package.json#peerdependencies)?